avm/res/network/private-endpoint/tests/e2e/defaults/main.test.bicep

```
@@ -0,0 +1,63 @@
+targetScope = 'subscription'
+
+metadata name = 'Using only defaults'
+metadata description = 'This instance deploys the module with the minimum set of required parameters.'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'network.privateendpoints-${serviceShort}-rg'
```

```suggestion
param resourceGroupName string = 'network.privateendpoints-${namePrefx}-${serviceShort}-rg'
```
Just a reminder to add the `namePrefix` param reference here.

---

Done with a different convention as discussed